config/locales/views/activities/nl.yml

```
@@ -126,3 +126,9 @@ nl:
     series_activities_table:
       course_has_no_rights_with_fix: Deze cursus heeft niet de rechten om deze privé-oefening te gebruiken. Klik hier om deze rechten te geven.
       course_has_no_rights_with_contact: Deze cursus heeft niet de rechten om deze privé-oefening te gebruiken. Contacteer de auteur of verwijder deze oefening uit de cursus.
+    coding_scratchpad:
+      title: "Voer %{programming_language} code uit in je browser"
+      experimental_feature_html: Deze experimentele toepassing laat je toe om code te schrijven en uit te voeren in je browser, zodat je meteen kan beginnen. Suggesties en problemen kan je melden bij <a href="%{url}">team Dodona</a>.
+      close: Sluiten
+      start_coding: Begin met coderen
```

```suggestion
      start_coding: Code uitvoeren
```
In the future, we would use "Kladblok" and "Scratchpad" as label text + add some kind of run icon.